api/pkg/cli/formatter/field.go

```diff
@@ -73,6 +74,22 @@ func FormatTags(tags []*client.TagResponseBody) string {
 	return sb.String()
 }
 
+// FormatPlatforms returns list of platforms seperated by comma
+func FormatPlatforms(platforms []*client.PlatformResponseBody) string {
+	var sb strings.Builder
+	if len(platforms) == 0 {
```

Comment:
Just a thought, as according to the [proposal](https://github.com/tektoncd/community/blob/main/teps/0070-tekton-catalog-task-platform-support.md#proposal) if the annotation for platform is not specified, `linux/amd64` platform is meant by default right, should we add `linux/amd64` instead of `---`

Reply:
hmm, interesting idea, actually. At this moment this logic (no platform = `linux/amd64` platform) is implemented on hub resource source code parsing level, so expectation is that all resources in DB already have `linux/amd64` if their source code doesn't have platform annotation. 
So I'd keep it as is, because substitution logic is already implemented on the backend. And if there are no platforms, it means that hub deployment doesn't have latest code and showing `linux/amd64` for all tasks can be incorrect information.  What do you think?

Reply:
Yes I think let's keep it from what we get from backend instead on manually adding it, since resources with no platform already have `linux/amd64` in db, so the above implementation looks fine to me.
Thanks